### ISSUE TITLE
Tensorflow v2 Compatibility issue

### ISSUE THREAD

**Issue by emersonard** [Azure/MachineLearningNotebooks#736]
In the [Train Hyperparameter Tune Deploy with Tensorflow](https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/ml-frameworks/tensorflow/deployment/train-hyperparameter-tune-deploy-with-tensorflow/train-hyperparameter-tune-deploy-with-tensorflow.ipynb) tutorial, there are v1/v2 compatibility issues with the stock AzureML notebook environment.

Cell 23 "Predict on the test set" returns an attribute error, no function import_meta_graph. Fixed by replacing standard tensorflow import with:

```javascript
import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()
```

**Comment by emersonard** [comment]
Update: Cell 14 output
`WARNING - framework_version is not specified, defaulting to version 1.13.`

**Comment by ram-msft** [comment]
@emersonard Thanks for the feedback. We will update the notebook shortly.

**Comment by lostmygithubaccount** [comment]
@swatig007 is this still an issue for this notebook?

**Comment by swatig007** [comment]
this issue is resolved
